Modules/Module_Processing/Module_Population/xASL_stat_GetDICOMStatistics.m

```diff
@@ -36,78 +39,55 @@ function xASL_stat_GetDICOMStatistics(x, ScanType, HasSessions, bOverwrite)
     HasSessions = false;
 end
 
-% Mat fields that will be printed (some are in x.(field) and some in x.Q.(field))
-matFields = {'RepetitionTime' 'EchoTime' 'NumberOfTemporalPositions' 'MRScaleSlope' 'RescaleSlopeOriginal' 'RescaleIntercept' 'Scanner' 'SliceReadoutTime'};
-nFields = length(matFields);
 PathTSV = fullfile(x.D.DICOMparameterDir, ['QuantificationParameters_' ScanType '.tsv']);
 
 % Print header
 TSV = {'participant_id' 'session'};
-TSV(1,3:2+nFields) = matFields(1:nFields);
 
-% Initialize empty table
-numElements = size(TSV,2);
-numSubjectsSessions = x.dataset.nSubjects*x.dataset.nSessions;
-for iSubjSess=1:numSubjectsSessions
-    TSV(1+iSubjSess,:) = repmat({' '},1, numElements);
-end
 
-%% 1) Load & save individual parameter files
-fprintf('%s\n','Loading & saving individual parameter files...  ');
+%% 1. Loading JSON sidecars
+fprintf('%s\n', ['Loading ' ScanType '.json sidecars...  ']);
 
 for iSubject=1:x.dataset.nSubjects
     for iSession=1:x.dataset.nSessions
 
         % Track progress
         iSubjSess = (iSubject-1)*x.dataset.nSessions+iSession;
-        xASL_TrackProgress(iSubjSess, x.dataset.nSubjects * x.dataset.nSessions);        
+        xASL_TrackProgress(iSubjSess, x.dataset.nSubjects * x.dataset.nSessions);
         
         % Default parameters
         TSV{1+iSubjSess,1} = x.SUBJECTS{iSubject};
         TSV{1+iSubjSess,2} = x.SESSIONS{iSession};
         
-        TSV(1+iSubjSess, 3:nFields+2) = repmat({NaN}, [1 nFields]);
-        x.S.par(iSubjSess,1:nFields) = NaN;
+        % TSV(1+iSubjSess, 3:nFields+2) = repmat({NaN}, [1 nFields]);
```

Comment:
Agree with the edit. But I would completely remove those. Did it myself already.